hit-test.html

```diff
@@ -146,6 +149,7 @@
       }
 
       function onEndSession(session) {
+        xrHitTestSource.cancel();
```

Comment:
Curious as to why this is beneficial? I'd imagine that the hit test sources don't do anything once the session is ended?

Reply:
Mostly educational - I do not want people to just drop all references w/o canceling hit test sources. In this particular case it does not really matter.